src/pages/Gallery/Gallery.tsx

```diff
@@ -1,7 +1,118 @@
-import React from "react"
+import React, { useState } from "react"
+
+import styles from "./Gallery.module.scss"
+import classnames from "classnames"
+
+const Lines = () => {
+  return (
+    <div className={styles.lines}>
+      {Array.from({ length: 6 }).map((_, index) => (
+        <div key={index} className={styles.line} />
+      ))}
+    </div>
+  )
+}
 
 const Gallery = () => {
-  return <h1>Wojak gallery here</h1>
+  const [isFilterOpen, setIsFilterOpen] = useState(false)
```

Comment:
This shouldn't be here I think, it's possible we'll need to use same dropdown component elsewhere and it would be useful to have it in its own component in `components` folder